Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -283,8 +277,7 @@ namespace Camera
         AZ::TransformNotificationBus::Handler::BusDisconnect(m_entityId);
         CameraRequestBus::Handler::BusDisconnect(m_entityId);
 
-        auto atomViewportRequests = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
-        if (atomViewportRequests)
+        if (auto atomViewportRequests = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get())
```

Comment:
The `atomViewportRequests` variable isn't used, do you need to query the interface?

Reply:
Hmm yeah good point, I can just delete this entirely, the check before doesn't seem to be needed